File: udacity/cs253/Lesson02a_Templates/FizzBuzz.py
def get(n):
	fizzbuzz = []
	if n and n.isdigit():
		n = int(n)
		if n > 30:
			n = 30
		for i in range(1, n+1):
			if not i % 15:
				item = 'FizzBuzz'
			elif not i % 3:
				item = 'Fizz'
			elif not i % 5:
				item = 'Buzz'
			else:
				item = str(i)
			fizzbuzz.append(item)
	return fizzbuzz

File: udacity/cs253/Integration/_01-04_with_vuejs/static/fizzbuzz.js
var app = new Vue({
	el: '#fizzbuzz',
	data: {
		num: null,
		FB: []
	},
	methods: {
		getFB: function(){
			var FB = []
			var fb = ''
			var len = Number(this.num)
			for(let i = 1; i <= len; i++){
				if (i % 15 === 0){
					fb = 'FizzBuzz'
				}else if(i % 3 === 0){
					fb = 'Fizz'
				}else if(i % 5 === 0){
					fb = 'Buzz'
				}else{
					fb = i
				}
				FB.push(fb)
			}
			this.FB = FB
			this.show = !this.show
			console.log(FB)
		}
	}
})
